src/components/Alert/Alert.tsx

```diff
@@ -69,9 +68,9 @@ const Alert = ({
         <h2 sx={{ variant: "text.heading2", margin: 0, paddingBottom: 3 }}>
           {title}
         </h2>
-        <p sx={{ variant: "text.body1", margin: 0, color: "text.40" }}>
+        <div sx={{ variant: "text.body1", margin: 0, color: "text.40" }}>
```

Comment:
The default margin isn't 0 for div tags?

Reply:
It is, I'll remove the margin style, good catch
I changed it to a div because the Alert examples were using `<p>` tags in the children, and `<p>` can't have `<p>` as children

```suggestion
        <div sx={{ variant: "text.body1", color: "text.40" }}>
```